packages/xo-server/src/xapi-object-to-xo.mjs

```diff
@@ -746,7 +746,13 @@ const TRANSFORMS = {
   task(obj) {
     let applies_to
     if (obj.other_config.applies_to) {
-      applies_to = obj.$xapi.getObject(obj.other_config.applies_to, undefined).uuid
+      const object = obj.$xapi.getObject(obj.other_config.applies_to, undefined)
+      if (object === undefined) {
+        debug(
+          `Undefined object while trying to get object : ${obj.other_config.applies_to} (linked to object ${obj.$id})`
+        )
+      }
+      applies_to = object?.uuid
```

Comment:
It is not unexpected that the object would not exist as can be seen by the `undefined` value passed as second parameter to `getObject` which is here explicitly to handle this case.

But it's indeed a bug because the original code did not handle this case when reading the `uuid` property.

Also, just an idea, debugs are not visible by default, maybe it would be better to assign the original value (opaque ref) to `applies_to` instead?